native/packages/react-native-bpk-component-button/src/layout/BpkButtonContainer.android.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  View,
+  TouchableNativeFeedback,
+} from 'react-native';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  getStyleForElement,
+  getAndroidBackgroundColour,
+  iconPropType,
+  themePropType,
+} from './../BpkButton-utils';
+
+const BpkButtonContainer = (props) => {
+  const {
+     type,
+    title,
```

Comment:
A lot of these props aren't used? I am surprised the lint passed.

(`icon` & `iconOnly` for instance)

Reply:
that's a good edge case for the lint then, you're absolutely right, I've copy-pasted from the main file, lint was passing so I thought it was all good